src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,164 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: A consolidated resource for methods a developer can use to test templates
+
+There are many ways you can test your Koji templates.
```

Comment:
This sentence doesn't provide much information or context. In general, this intro could set the stage a bit better, as well as to highlight that we have powerful testing tools built into the Koji platform.